src/trans/external/specnorm.F90

```diff
@@ -64,11 +64,11 @@ SUBROUTINE SPECNORM(PSPEC,KVSET,KMASTER,KRESOL,PMET,PNORM)
 ! Declaration of arguments
 
 
+REAL(KIND=JPRB)             , INTENT(OUT) :: PNORM(:)
```

Comment:
We might as well make the corresponding argument in SPNORM_CTL mandatory as well.

Reply:
done